OSPSuite.Presentation.Importer/Core/IUnformattedData.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using DevExpress.DataProcessing;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   public interface IUnformattedData
+   {
+      IEnumerable<string> GetRow(int index);
+      IEnumerable<string> GetColumn(int columnIndex);
+      IEnumerable<string> GetColumn(string columnName);
+      Dictionary<string, ColumnDescription> Headers { get; }
+
+      IEnumerable<IEnumerable<string>> GetRows(Func<IEnumerable<string>, bool> filter);
+
+      bool AddRow(IEnumerable<string> row);
+      void AddColumn(string columnName, int columnIndex);
+      DataTable GetSheetAsDataTable();
+   }
+
+   public class UnformattedData : IUnformattedData
+   {
+      private readonly List<List<string>> _rawDataTable = new List<List<string>>();
+
+      public Dictionary<string, ColumnDescription> Headers { get; set; } =
+         new Dictionary<string, ColumnDescription>(); //we have to ensure headers and RawData sizes match
+
+      public void AddColumn(string columnName, int columnIndex) //it seems to me there is little sense in adding column after column
+         //the list of headers is somehow the definition of the table
+      {
+         Headers.Add(columnName, new ColumnDescription(columnIndex));
+      }
+
+      public void CalculateColumnDescription(List<ColumnDescription.MeasurementLevel> levels)
+      {
+         Headers.ForEach(header =>
+         {
+            if (header.Value.Level == ColumnDescription.MeasurementLevel.NotSet) //hell, we could even not check here
+               header.Value.Level = levels[header.Value.Index];
+         });
+      }
+
+      public bool AddRow( IEnumerable<string> row)
```

Comment:
What does the boolean does as returned value.
In fact, what does that mean if it's false? Should not we throw an exception instead?